src/components/widgets/outputs/OutputsCard.vue

```diff
@@ -6,6 +6,43 @@
     layout-path="dashboard.outputs-card"
     menu-breakpoint="lg"
   >
+    <template #menu>
+      <v-menu
+        bottom
+        left
+        offset-y
+        transition="slide-y-transition"
+        :close-on-content-click="false"
+      >
+        <template #activator="{ on, attrs }">
+          <v-btn
+            fab
+            x-small
+            text
+            v-bind="attrs"
+            class="ms-1 my-1"
+            v-on="on"
+          >
+            <v-icon>
+              $cog
+            </v-icon>
+          </v-btn>
+        </template>
+
+        <v-list dense>
+          <v-list-item @click="showHidden = !showHidden">
+            <v-list-item-action class="my-0">
+              <v-checkbox :input-value="showHidden" />
+            </v-list-item-action>
+            <v-list-item-content>
+              <v-list-item-title>
+                Show Hidden
```

Comment:
Strings like this need to be localizable, so it should move to "en.yaml" and consumed with `$t(app....)` so it can be translated

Reply:
Lets also make this "Show hidden outputs" to make it more descriptive and distinct from any other.

Reply:
agree

Reply:
ok, got the other things done. where do you recomend putting this in the translation files?